app/models/article.rb

```diff
@@ -6,7 +6,7 @@ class Article < ApplicationRecord
 
   acts_as_taggable_on :tags
 
-  attr_accessor :publish_under_org
+  attr_accessor :publish_under_org, :new_author
```

Comment:
Is `new_author` necessary vs using `user_id`?

Reply:
yep good call